src/mumble/Themes.cpp

```diff
@@ -8,6 +8,14 @@
 #include "MumbleApplication.h"
 #include "Global.h"
 
+#include <QFile>
+#include <QGuiApplication>
```

Comment:
Please also double check (at least when we are done here), if all of those includes are actually needed.

Reply:
I think only these includes are needed "QPalette, QSettings, QStyleHints", but not 100% sure.